get_forecast.py

```diff
@@ -131,19 +143,36 @@ def send_sns(boto3_session, sns_arn, message):
 
 
 def display_output(boto3_session, message):
-    secrets_manager_client = boto3_session.client('secretsmanager')
+    secrets_manager_client = None
+    secret = ""
+    if 'SECRETS_MANAGER_ENDPOINT' in os.environ:
+        logger.info("Using secrets_manager_endpoint.")
+        secrets_manager_client = boto3_session.client('secretsmanager', os.environ['SECRETS_MANAGER_ENDPOINT'])
+    else:
+        secrets_manager_client = boto3_session.client('secretsmanager')
+
+    try:
+        secret = get_secret(secrets_manager_client)
+    except Exception as e:
+        logger.error("get_secret failed")
+
     try:
-        slack_url='https://' + get_secret(secrets_manager_client, SLACK_SECRET_KEY_NAME)
+        slack_url = json.loads(secret)[SLACK_SECRET_KEY_NAME]
         send_slack(slack_url, message)
     except Exception as e:
-        logger.info("Disabling Slack, URL not found")
+        logger.info("Disabling Slack, URL not found", e)
```

Comment:
lets discuss why print e here, if you want it for debugging add 
logger.debug(e) 

but it just makes logs messy